src/spikeinterface/postprocessing/spike_locations.py

```diff
@@ -44,13 +50,28 @@ def _run(self, **job_kwargs):
         uses the`sortingcomponents.peak_localization.localize_peaks()` function to triangulate
         spike locations.
         """
-        from spikeinterface.sortingcomponents.peak_localization import localize_peaks
+        from spikeinterface.sortingcomponents.peak_localization import _run_localization_from_peak_source
 
         job_kwargs = fix_job_kwargs(job_kwargs)
 
         we = self.waveform_extractor
 
-        spike_locations = localize_peaks(we.recording, self.spikes, **self._params, **job_kwargs)
+        extremum_channel_inds = get_template_extremum_channel(we, peak_sign="neg", outputs="index")
+
+        params = self._params.copy()
+        channel_from_template = params.pop("channel_from_template")
+
+        # @alessio @pierre: where do we expose the parameters of radius for the retriever (this is not the same as the one for locatization it is smaller) ???
+        spike_retriever = SpikeRetriever(
+            we.recording,
+            we.sorting,
+            channel_from_template=channel_from_template,
+            extremum_channel_inds=extremum_channel_inds,
+            radius_um=50,
+            peak_sign=self._params.get("peaks_sign", "neg"),
```

Comment:
This neg should not be hardcoded, and peak_sign should be an argument of compute_spike_locations